flink-runtime/src/test/java/org/apache/flink/runtime/util/JvmExitOnFatalErrorTest.java

```diff
@@ -95,12 +94,14 @@
     @Test
     public void testExitJvmOnOutOfMemory() throws Exception {
         // this test works only on linux
-        assumeTrue(OperatingSystem.isLinux());
+        //        assumeTrue(OperatingSystem.isLinux());
```

Comment:
maybe replace this with `assumeFalse(OperatingSystem.isWindows());`

Reply:
Yes, definitely. Good catch!